src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -456,6 +461,45 @@ protected function move(Request $request, $movement)
     }
 
     /**
+     * @param Request $request
```

Comment:
I think sth i still wrong with rebase and there will be not such a diff in final version of this PR :smile: